draft-foudil-securitytxt.md

```diff
@@ -285,10 +284,22 @@ Example of an OpenPGP key being referenced by its fingerprint:
 Encryption: openpgp4fpr:5f2de5521c63a801ab59ccb603d49de44b29100f
 ~~~~~~~~~~
 
+### Expires {#expires}
+
+This field indicates the date/time after which the data contained in the "security.txt"
+file is considered stale and should not be used.
+The value of this field follows the format defined in section 3.3 of {{!RFC5322}}.
+
+This field MUST NOT appear more than once.
+
+~~~~~~~~~~
+Expires: Thu, 31 Dec 2020 18:37:07 -0800
```

Comment:
Is the date arbitrary? Is it the implementer's sole discretion to set a realistic expiration date? How does one determine what a suitable expiration date is?

Reply:
I added a reference to the staleness section from the security considerations section